repology/parsers/parsers/aosc.py

```diff
@@ -56,6 +59,8 @@ def iter_parse(self, path: str, factory: PackageFactory) -> Iterable[PackageMake
                 pkg.add_categories(pkgdata['pkg_section'], pkgdata['section'])
                 pkg.set_summary(pkgdata['description'])
 
+                pkg.add_links(LinkType.PACKAGE_SOURCES, pkgdata['srcurl'])
```

Comment:
`PACKAGE_SOURCES` are for downstream package, e.g. package recipe. You need `UPSTREAM_DOWNLOAD` or `UPSTREAM_REPOSITORY` here based on `srctype` value.